.github/workflows/build.yml

```diff
@@ -77,6 +77,20 @@ jobs:
       if: ${{ runner.os != 'Windows' }}
       run: ./eng/common/cibuild.sh -configuration Release -prepareMachine
 
+    - name: Attest artifacts
+      uses: actions/attest-build-provenance@951c0c5f8e375ad4efad33405ab77f7ded2358e4 # v1.1.1
+      if: |
+        runner.os == 'Windows' &&
+        github.event.repository.fork == false &&
+        (github.ref_name == github.event.repository.default_branch ||
+         startsWith(github.ref, 'refs/heads/dev') ||
+         startsWith(github.ref, 'refs/heads/rel/') ||
+         startsWith(github.ref, 'refs/tags/'))
+      with:
+        subject-path: |
+          ./artifacts/bin/AspNet.Security.OAuth.*/Release/**/AspNet.Security.OAuth.*.dll
```

Comment:
Do we need to be ultra specific here or a more generic filter like `./artifacts/bin/**/Release/**/*.dll` could work too?

Reply:
This avoids attesting anything that's a dependency that we didn't build ourselves and avoids the sample site too.

Reply:
Makes sense 👍🏻